entrypoint.py

```diff
@@ -36,23 +40,34 @@ def setup_env(
 def install_edr(adapter: str, project_dir: Optional[str]):
     logging.info("Getting Elementary dbt package version.")
     try:
-        dbt_pkg_ver = (
-            subprocess.run(
-                [
-                    "dbt",
-                    "-q",
-                    "run-operation",
-                    "get_elementary_dbt_pkg_version",
-                    "--project-dir",
-                    "/edr_stager_dbt_project",
-                ],
-                check=True,
-                capture_output=True,
-                cwd=project_dir,
-            )
-            .stdout.decode()
-            .strip()
-        )
+        dbt_pkg_ver = None
+        command_results = subprocess.run(
+            [
+                "dbt",
+                "--log-format",
+                "json",
+                "run-operation",
+                "get_elementary_dbt_pkg_version",
+                "--project-dir",
+                "/edr_stager_dbt_project",
+            ],
+            check=True,
+            capture_output=True,
+            cwd=project_dir,
+        ).stdout.decode("utf-8")
+
+        for log_line in command_results.splitlines():
+            try:
+                log = json.loads(log_line)
+                message = log.get("info", {}).get("msg") or log.get("data", {}).get(
+                    "msg"
+                )
+                if message.startswith(EDR_STAGER_PREFIX):
+                    dbt_pkg_ver = message[len(EDR_STAGER_PREFIX) :]
+                    break
+            except Exception:
+                pass
```

Comment:
Could you just add a `logging.error()` here please?

Reply:
Yeah!

Reply:
Done